parity/params.rs

```diff
@@ -256,6 +257,7 @@ impl Default for GasPricerConfig {
 		GasPricerConfig::Calibrated {
 			usd_per_tx: 0.0001f32,
 			recalibration_period: Duration::from_secs(3600),
+			api_endpoint: "https://api.etherscan.io/api?module=stats&action=ethprice".to_owned()
```

Comment:
Can't we use `ETHERSCAN_ETH_PRICE_ENDPOINT` here?

Reply:
Done